tests/unit/Landing.spec.js

```
@@ -0,0 +1,61 @@
+      name: 'ReleaseNote',
+      component: ReleaseNote
+    },
+    {
+      path: '/feedback',
+      name: 'Feedback',
+      component: Feedback
+    }
+  ]
+})
+
+describe('Landing.vue', () => {
```

ESLint

[error] 'describe' is not defined.

(no-undef)